packages/blade/docs/tokens/Breakpoints.stories.mdx

```diff
@@ -34,21 +35,40 @@ import { paymentTheme, bankingTheme } from '../../src/tokens';
 
 # 📏 Breakpoints
 
+export const breakpointScreenMapping = {
+  base: 'Base styles',
+  xs: 'Small Mobiles',
+  s: 'Mobiles and Small Tablets',
+  m: 'Medium and Large Tablets <br/>(This breakpoint can be used for desktop styling)',
+  l: 'Medium Desktop',
+  xl: 'HD Desktop',
+};
+
 export const Breakpoints = () => {
   const { theme } = useTheme();
   return (
     <table>
       <thead>
         <tr>
           <th>Token Name</th>
-          <th />
+          <th>Token Value</th>
+          <th>Corresponding Media Query</th>
+          <th>Corresponding Screen</th>
```

Comment:
Added corresponding media query that we use and the screen size that it relates to.

<img width="1161" alt="image" src="https://user-images.githubusercontent.com/30949385/222670790-66629051-bb5b-49ea-bdcf-a6a6fada61da.png">

https://saurabh-box-docs--61c19ee8d3d282003ac1d81c.chromatic.com/?path=/story/tokens-breakpoints--page&globals=measureEnabled:false

Reply:
worth adding: the little usage of how to use breakpoint hook from blade utils in the same page under breakpoints?

Reply:
added in upcoming commit